.devcontainer/scripts/configure-proxies.sh

```diff
@@ -95,6 +96,21 @@ if [ "${USE_PROXIES}" = "true" ]; then
     # enable root user to "apt-get" via proxy
     echo "Acquire::http::proxy \"${HTTP_PROXY}\";" >> /etc/apt/apt.conf
     echo "Acquire::https::proxy \"${HTTPS_PROXY}\";" >> /etc/apt/apt.conf
+
+    # apply proxy for docker config
+    DOCKER_CONFIG_INITIAL_CONTENT="{}"
+    mkdir -p /home/${USERNAME}/.docker
```

Comment:
I suggest we introduce a variable for `/home/${USERNAME}/.docker` and re-use that also for `CONFIG_FILE`